civil-servants.md

```diff
@@ -0,0 +1,48 @@
+# Signalen for civil servants
+
+We're rebuilding Signalen so that municipalities everywhere can benefit.
```

Comment:
Maybe replace "rebuilding" with "enhancing" or similar, because the effort is not starting over, it's building on top of the work that has already been done.

Reply:
In civil-servants.md, I've written:
"### Reliability

Amsterdam's Signalen implementation has a high uptime. We believe an even higher percentage is possible. We're currently rearchitecting Signalen to make it easier to maintain and troubleshoot."

Is this accurate?